### PR TITLE
Node18 patch arique

### LINKED ISSUE #3: Texter should get text that their list is ready
As a user, once a person has assigned me a list of people to text, I should get a text notification letting me know my list is ready and a link to the spoke app to get started.

### LINKED ISSUE #3: Nodejs 20 Upgrade
Update the Nodejs version to 18 to avoid Twilio deprecation.
Stephanie & Daniel

### PR DESCRIPTION
**Pull request by Arique1104**
# Fixes https://github.com/StateVoicesNational/Spoke_Hackathon_Fork/issues/3

## Description

_Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change, and any blockers that make your change a WIP_

# Checklist:

- [ ] I have manually tested my changes on desktop and mobile
- [ ] The test suite passes locally with my changes
- [ ] If my change is a UI change, I have attached a screenshot to the description section of this pull request
- [ ] [My change is 300 lines of code or less](https://github.com/MoveOnOrg/Spoke/blob/main/CONTRIBUTING.md#submitting-your-pull-request), or has a documented reason in the description why it’s longer
- [ ] I have made any necessary changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] My PR is labeled [WIP] if it is in progress

### CODE REVIEW

File: lambda.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
-"use strict";
-const AWS = require("aws-sdk");
+"use strict";;
```
Comment (Arique1104) [inline lambda.js]: Interesting.  I should remove that extra semi-colon that came on after the migration commands.

File: package.json [hunk 2]
```diff
@@ -4,8 +4,8 @@
   "description": "Spoke",
   "main": "src/server",
   "engines": {
-    "node": ">=16.18.0",
-    "npm": "8.19.2"
+    "node": ">=20.0.0",
```
Comment (Arique1104) [inline package.json:7]: This was a critical lesson learned!  Some of our features do have have support in version 21, and so choosing version20 is our main priority, but no higher!

File: src/extensions/contact-loaders/csv-s3-upload/index.js [hunk 3]
```diff
@@ -3,7 +3,8 @@ import { unzipPayload } from "../../../workers/jobs";
 import { getConfig, hasConfig } from "../../../server/api/lib/config";
 import { gunzip } from "../../../lib";
 
-import AWS from "aws-sdk";
+import { getSignedUrl } from "@aws-sdk/s3-request-presigner";
+import { PutObjectCommand, S3 } from "@aws-sdk/client-s3";
```
Comment (Arique1104) [inline src/extensions/contact-loaders/csv-s3-upload/index.js:7]: Why wasn't something similar added to mobile-commons document up top?

File: src/extensions/action-handlers/mobilecommons-signup.js [hunk 4]
```diff
@@ -1,5 +1,4 @@
 import request from "request";
-import aws from "aws-sdk";
```
Comment (Arique1104) [inline src/extensions/action-handlers/mobilecommons-signup.js:2]: I've reviewed this document and see no aws method being used or replaced.

**Comment by mau11** [comment]
Closing as Spoke has been updated to Node 20

**Pull request closed without merging**